src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,90 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+interface Props {
+  selectedTodo: Todo | null;
+  closeTodo: () => void;
+}
+
+export const TodoModal: React.FC<Props> = ({ selectedTodo, closeTodo }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isError, setIsError] = useState(false);
+
+  const loadData = async () => {
+    try {
+      const currentUser = await getUser(selectedTodo?.userId || 0);
+
+      setUser(currentUser);
+    } catch {
+      setIsError(true);
+    }
+  };
+
+  useEffect(() => {
+    loadData();
+  }, [selectedTodo]);
+
+  const isCompleted = selectedTodo?.completed
+    ? 'Done'
+    : 'Planned';
 
-export const TodoModal: React.FC = () => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
+      {!user
+        ? (
+          <Loader />
+        ) : (
+          <div className="modal-card">
+            <header className="modal-card-head">
+              <div
+                className="modal-card-title has-text-weight-medium"
+                data-cy="modal-header"
+              >
+                {`Todo #${selectedTodo?.id}`}
+              </div>
 
-      {true ? (
-        <Loader />
-      ) : (
-        <div className="modal-card">
-          <header className="modal-card-head">
-            <div
-              className="modal-card-title has-text-weight-medium"
-              data-cy="modal-header"
-            >
-              Todo #2
-            </div>
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
let's fix this eslint issue:smiley:

try to give `aria-label` to button, i think it can help